src/main/java/io/github/tryexceptelse/jdex/Settings.java

```diff
@@ -3,8 +3,4 @@
 /**
  * Settings object to store program settings
  */
-class Settings {
-    String autoSaveLoc;
-    String defaultSaveLoc;
-    String lastSavedFile;
-}
+class Settings {} // skeleton
```

Comment:
What methods are going to be in this class? What will it be calling and what will it be returning? I see it used to be methods returning Strings, and that doesn't make much sense to me.